lib/betterdoc/containerservice/engine.rb

```diff
@@ -0,0 +1,6 @@
+module Betterdoc
+  module Containerservice
+    class Engine < ::Rails::Engine
```

Comment:
Just for understanding: what does this class do?

Reply:
It just tells Rails that this gem is actually Rails engine. Meaning that it should use its routes, views, controllers, ... and whatever is present there.

More info here https://guides.rubyonrails.org/engines.html